OpenRA.Game/ObjectCreator.cs

```diff
@@ -54,7 +55,13 @@ public ObjectCreator(Manifest manifest, FileSystem.FileSystem modFiles)
 				Assembly assembly;
 				if (!ResolvedAssemblies.TryGetValue(hash, out assembly))
 				{
-					assembly = Assembly.Load(data);
+					Stream debugStream = null;
+					if (Type.GetType("Mono.Runtime") != null && modFiles.TryOpen(path + ".mdb", out debugStream))
+						assembly = Assembly.Load(data, debugStream.ReadAllBytes());
+					else if (modFiles.TryOpen(path.Substring(0, path.Length - 4) + ".pdb", out debugStream))
```

Comment:
@atlimit8 Your first line is wrong as it is the opposite logic of mine and I test my changes before committing.

Yes, @pchote's version is probably easier to read so I used that with a small change: `isMonoRuntime` is initialized by a field initializer instead of duplicating it to both constructors.